python-api/gstswitch/connection.py

```diff
@@ -282,6 +282,29 @@ def set_composite_mode(self, mode):
             new_message = "{0}: {1}".format(message, "set_composite_mode")
             raise ConnectionError(new_message)
 
+    def get_composite_mode(self):
+        """get_composite_mode(out b result);
+        Calls get_composite_mode remotely
+
+        :returns: tuple with first element being the current compsition mode
+        """
+        try:
+            connection = self.connection
+            result = connection.call_sync(
+                self.bus_name,
+                self.object_path,
+                self.default_interface,
+                'get_composite_mode',
+                None,
+                GLib.VariantType.new("(i)"),
+                Gio.DBusCallFlags.NONE, -1,
+                None)
+            return result
```

Comment:
Looks like you're working from an older version of this file. These should probably be mapped to the names at https://github.com/timvideos/gst-switch/blob/master/python-api/gstswitch/controller.py#L21 in some way?